sklearn/utils/testing.py

```diff
@@ -796,6 +796,20 @@ def _get_args(function, varargs=False):
         return args
 
 
+def _get_parent_args(Estimator):
+    """Helper to get list of parents properties"""
+    # NOTE this works only in python3.5
+    if sys.version_info < (3, 5):
+        NotImplementedError("_get_parent_args is not available "
```

Comment:
It is actually not supported by the backport too (`sklearn.externals.funcsigs.signature`)
I got the following error :( `callable <slot wrapper '__init__' of 'object' objects> is not supported by signature`